provider-profile-service/src/main/java/team/proximity/provider_profile_service/payment_method/PaymentMethodServiceImpl.java

```diff
@@ -24,12 +26,14 @@ public class PaymentMethodServiceImpl implements PaymentMethodService {
     private final PaymentMethodFactory paymentMethodFactory;
     private final PaymentMethodRepository paymentMethodRepository;
     private final PaymentPreferenceRepository paymentPreferenceRepository;
+    private final PaymentMethodValidatorFactory validatorFactory;
 
-    public PaymentMethodServiceImpl(PaymentMethodMapper paymentMethodMapper, PaymentMethodFactory paymentMethodFactory, PaymentMethodRepository paymentMethodRepository, PaymentPreferenceRepository paymentPreferenceRepository) {
+    public PaymentMethodServiceImpl(PaymentMethodMapper paymentMethodMapper, PaymentMethodFactory paymentMethodFactory, PaymentMethodRepository paymentMethodRepository, PaymentPreferenceRepository paymentPreferenceRepository, PaymentMethodValidatorFactory validatorFactory) {
```

Comment:
Accepts too many arguments, refactor it

Reply:
Do you think it will be best to use lombok require argument constructor annotation. i had wanted to spend some time looking at its disadvantage before using it